setup.py

```diff
@@ -46,7 +46,7 @@
         #       and ray[rllib] under [train] it  prevents rllib from getting installed.
         #       For simplicity we just install both here. In the future we may want to
         #       address this bug head on to keep our SMARTS base install more lean.
-        "ray[rllib]==0.8.6",  # We use Ray for our multiprocessing needs
+        "ray[rllib]==1.0.1.post1",  # We use Ray for our multiprocessing needs
```

Comment:
1.1.0 is published. Shall we use this instead?

Reply:
1.1.0 seems to be a bit new, (Dec 24th). It might be reasonable given another week.

Reply:
Sure. `1.1.0` also might have breaking changes.

Reply:
While that is true, I am more worried about how well it has been tested.